crates/re_renderer/shader/generic_skybox.wgsl

```diff
@@ -14,11 +14,37 @@ fn skybox_light_srgb(dir: vec3f) -> vec3f {
     return vec3f(0.85) + 0.15 * rgb;
 }
 
+// -----------------------------------------------
+// Adapted from
+// https://www.shadertoy.com/view/llVGzG
+// Originally presented in:
+// Jimenez 2014, "Next Generation Post-Processing in Call of Duty"
```

Comment:
```suggestion
// Jimenez 2014, "Next Generation Post-Processing in Call of Duty"
//
// A good overview can be found in
// https://blog.demofox.org/2022/01/01/interleaved-gradient-noise-a-different-kind-of-low-discrepancy-sequence/
```

Reply:
ah nice blog post. Tbh I just went straight to shadertoy to pick something that fits the bill and didn't do as much research 😄